index.html

```diff
@@ -1,192 +1,192 @@
-                    <img class="language-icon" src="img/js.png" alt="javascript">
-                    <p class="name">Javascript</p>
-                </li>
-                <li class="skills-parent">
-                    <img class="language-icon" src="img/html.png" alt="html">
-                    <p class="name">HTML</p>
-                </li>
-                <li class="skills-parent">
-                    <img class="language-icon" src="img/css.png" alt="css">
-                    <p class="name">CSS</p>
-                </li>
-            </ul>
-            </li>
-            <li class="languages separator">
-                <h3 class="language-title">Frameworks</h3>
-                <i class="arrow right"></i>
-            </li>
-            <li class="languages separator">
-                <h3 class="language-title">Skills</h3>
-                <i class="arrow right"></i>
-            </li>
-            <li class="languages separator">
-                <h3 class="language-title"></h3>
-            </li>
-        </ul>
-
         
-    </section>
-    </section>
-    <section class="contact">
-        <h2 class="title-contact">Contact me</h2>
-        <img class="text-image">
-        <p class="text-contact">If you have an application you are interested in developing, a feature that you need built or a
-            project that needs coding. I’d love to help with it </p>
-        <form action="https://formspree.io/f/mwkjbodb" method="POST">
-            <input type="text" name="name" id="name" placeholder="Name" maxlength="30" required>
-            <input type="email" name="email" id="email" placeholder="Email" required>
-            <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here" maxlength="500"
-                required></textarea>
-            <button type="submit"id="contact-button">Get in touch</button>
-           <p class="separator"></p>
-        </form>
-       
-    </section>
-    
-</body>
+        </section>
+        </section>
+        <section class="contact">
+            <h2 class="title-contact">Contact me</h2>
+            <!-- <img class="text-image"> -->
+            <p class="text-contact">If you have an application you are interested in developing, a feature that you need built or a
+                project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/mwkjbodb" method="POST">
+                <input type="text" name="name" id="name" placeholder="Name" maxlength="30" required>
+                <input type="email" name="email" id="email" placeholder="Email" required>
+                <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here" maxlength="500"
+                    required></textarea>
+                <button type="submit"id="contact-button">Get in touch</button>
+            </form>
+        </section>
+        
+    </body>
 </html>
```

Comment:
- Great work, @amiralda and @cancelei :clap: :clap: 
- Kindly consider these views and see how you can improve your portfolio

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|
![image](https://user-images.githubusercontent.com/99715640/222832522-f8f6c495-d793-4762-a6f6-42cadcf497be.png)|![image](https://user-images.githubusercontent.com/99715640/222832638-00a0a937-076c-404a-9144-74b5577a0925.png)|

Reply:
Here you can try overflow hidden property to fix BG. There are other commands that can apply for instance.